platform-sdk/swirlds-config-impl/src/main/java/com/swirlds/config/impl/converters/ConfigLevelConverter.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.config.impl.converters;
+
+import com.swirlds.config.api.converter.ConfigConverter;
+import com.swirlds.logging.api.internal.level.ConfigLevel;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import edu.umd.cs.findbugs.annotations.Nullable;
+
+public class ConfigLevelConverter implements ConfigConverter<ConfigLevel> {
```

Comment:
Javadoc

Reply:
I still do not like that place. We should move it to logging module. By doing so the converter need to be added "by hand" but we can do that easily in the logging classes.